### PR TITLE
Fixes #4673 Improvements in user management screen

### LINKED ISSUE #4673: Multiple Issues in account setting page
**What happened**:
When there are multiple invitations, accepting an invite causes the number of "Accept Invitation" buttons to increase.
Screenshot from 2024-05-24 16-32-45

**What you expected to happen**:
The number of "Accept Invitation" buttons should remain the same.

**What happened**:
When deleting a project invite or declining an invite , the dialog does not close after successful deletion.
Screenshot from 2024-05-24 16-33-23

**What you expected to happen**:
The dialog box should close after a successful invite deletion.

**What happened**:
Similarly, when leaving a project, the dialog box should close after successfully leaving the project.

Screenshot from 2024-05-24 16-33-13

**What you expected to happen**:
The dialog box should close after successfully leaving a project.

**What happened**:
When creating a user with an existing username, there is no error message displayed to inform the user that the username already exists.

**What you expected to happen**:
An error message should be displayed indicating that the username is already taken.

**How to reproduce it (as minimally and precisely as possible)**:
How to reproduce it (as minimally and precisely as possible):

Go to Settings.
1. Create at least three users.
2. Send invites to the users.
3. Try to accept an invite, delete an invite, and leave a project.

### PR DESCRIPTION
**Pull request by JanhaviAlekar**
fixes #4673 

## Proposed changes
commit 1: Fixed duplicate "Accept Invitation" button issue
commit 2: Fixed dialog box not closing after deleting an invite
commit 3: Fixed dialog box not closing after leaving a project
commit 4: Added toast error message to indicate that the username is already taken

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by hrishavjha — APPROVED** [review]
LGTM, thanks for the changes 🚀

File: chaoscenter/web/src/views/CreateNewUser/CreateNewUser.tsx [hunk 1]
```diff
@@ -41,6 +42,9 @@ export default function CreateNewUserView(props: CreateNewUserViewProps): React.
       {
         onSuccess: () => {
           handleClose();
+        },
+        onError: (err: any) => {
+          showError(err.errorDescription);
```
Comment (amityt) [inline chaoscenter/web/src/views/CreateNewUser/CreateNewUser.tsx:47]: Is this tested? Can you check if it is `err.message` or `err.description?`

**Review by amityt — APPROVED** [review]

File: chaoscenter/web/src/views/CreateNewUser/CreateNewUser.tsx (hunk 1 above)
Comment (JanhaviAlekar) [reply chaoscenter/web/src/views/CreateNewUser/CreateNewUser.tsx:47]: Yes, it's already tested. The err object does not have a field err.message; it is undefined. Instead, it has a field errDescription, which I used. 
Here is the err object as shown in the console:
error: "user_exists"
errorDescription:"This username is already assigned to another user"

**Review by amityt — APPROVED** [review]

**Comment by amityt** [comment]
Please resolve the merge conflicts @JanhaviAlekar

**Pull request merged**